Course_Part_09/src/Assignment38.java

```diff
@@ -0,0 +1,30 @@
+import java.util.HashMap;
+import java.util.Map;
+
+public class Assignment38 {
+    public static void main(String[] args) {
+
+        HashMap<Integer, String> shapesMap = new HashMap<>();
+        shapesMap.put(1, "Rectangle");
+        shapesMap.put(2, "Circle");
+        shapesMap.put(3, "Triangle");
+        shapesMap.put(4, "Square");
+        shapesMap.put(5, "Octagon");
+        shapesMap.put(6, "Ellipse");
+        System.out.println(shapesMap);
+
+        System.out.println();
+        shapesMap.forEach((Integer key, String value) -> System.out.println(key + "->" + value));
```

Comment:
Je ook mag de datatype van key en value weg laten waardoor je onderstaande code krijgt:

```
shapesMap.forEach((key, value) -> System.out.println(key + "->" + value));
```

(hebben we niet behandeld)